lib/trader-sdk/lib/indicators/bb-percent-b-mfi.ts

```diff
@@ -0,0 +1,71 @@
+import _ from 'lodash';
+import { BandWidth } from '../bollinger-bands';
+import { MFI } from '../money-flow-index';
+import { isWithinGivenBufferOrGreater } from '../util';
+
+const dependencies = {
+    BandWidth,
+    MFI
+};
+
+export interface PercentBMFIIndicatorArgs {
+    highPrices: number[],
+    lowPrices: number[],
+    closePrices: number[],
+    volumeAmounts: number[],
+    period?: number,
+    stdDev?: number
+};
```

Comment:
Codacy has a fix for the issue: [Unnecessary semicolon.](https://app.codacy.com/gh/dills122/trader-tools/pullRequest?prid=8102010)
```suggestion
}
```